### ISSUE TITLE
Potential accessibility issues with the Details (hidden text) component

### ISSUE THREAD

**Issue by JonathanCulling** [alphagov/govuk-frontend#3241]
## What
> We have recently completed some accessibility research with Assistive Technology users and found a couple of potential issues with the Details (hidden text) component.

## Why
> 1. Visually impaired participants using screen readers (such as NVDA and Read&Write) did not understand how they were supposed to interact with the links, as they were not announced as links or bullets.
> 2. Two of our cognitively impaired participants with health conditions complained that the large yellow highlighted active area could potentially trigger pressure headaches.
![GDS standard potential issues 3](https://user-images.githubusercontent.com/124280348/216372358-70c54146-466b-46c8-8a1e-d1bdd958226a.png)

## Anything else
> I have included a slide from our report which covers both of these issues. Are there any tweaks we can make to this component to eliminate these issues for our audience?

**Comment by querkmachine** [comment]
Hello! I'm going to move this over to our Frontend repo. We have an existing discussion about the Details component here that you may wish to contribute to: https://github.com/alphagov/govuk-design-system-backlog/issues/44

With regards to the first issue, I think our stance here is a little more defensive. We use the standardised `details`/`summary` HTML elements for this component—which have existed and been well supported for over a decade by now—and have done very little by way of functional modifications. They are, correctly, neither links nor buttons. 

That the screen readers tested fail to identify these to a user seems more like an oversight in the behaviour of the screen reader, rather than our component. If you're able to provide information on the screen readers involved we might be able to find more information on why this is and perhaps help them to resolve it.

**Comment by stevenjmesser** [comment]
Hi @JonathanCulling, are you able to provide more information on the screen readers involved in the testing please? I'll close this issue in 2 weeks' time if we don't hear back from you, but there's always the option to re-open it if needed.

**Comment by stevenjmesser** [comment]
Thanks for this, @JonathanCulling. Given the other issue posted (#3240) is regarding JAWS specifically, please can you confirm

- how many study participants reported links not being announced  
- which screen readers they were using, and
- that 11 participants in total were part of the study?

(I can't see a P11.) 

Our next steps would be to see whether other services, or our own testing, has surfaced these issues too.
